tests/jballerina-unit-test/src/test/resources/test-src/workers/sync-send.bal

```diff
@@ -629,6 +629,211 @@ public function testFailureForReceiveWithError() returns boolean {
     return x.w1 is E2 && x.w2 === true;
 }
 
+string appendCloneable = "";
+function testSimpleSyncSendWithCloneableExpression() {
+    string done = processWithCloneableExpression();
+    assertValueEquality("done", done);
+    assertValueEquality("w2cw2cw2cw2cw2cw1cw1cw1cw1cw1c", appendCloneable);
+}
+
+function processWithCloneableExpression() returns string {
+    worker w1Cloneable returns boolean {
+        value:Cloneable w1a = 10;
+        value:Cloneable w1b = 11;
+        value:Cloneable w1c = 12;
+        w1a -> w2Cloneable;
+        w1b ->> w2Cloneable;
+        w1c -> w2Cloneable;
+        foreach var i in 1 ... 5 {
+            appendCloneable = appendCloneable + "w1c";
+        }
```

Comment:
Do we need this? Aren't we just testing if the value is sent as expected (rather than if sync send is actually sync send)?

Reply:
Will remove foreach loops